src/Items/ItemShears.h

```diff
@@ -90,6 +90,14 @@ class cItemShearsHandler :
 			}
 		}
 	}
+
+
+
+	virtual float GetBlockBreakingStrength(BLOCKTYPE a_Block)
+	{
+		return !IsBlocksWeb(a_Block) && !IsBlockMaterialLeaves(a_Block) ? (IsBlocksWool(a_Block) ? 5.0f/* WOOL */ : super::GetBlockBreakingStrength(a_Block) /* none of them */) : 15.0f /* WEB and LEAVES */;
```

Comment:
It's actually very difficult to understand since it's a nested ternary operator and you'd need to know the exact operator precedence in C++ (which could be different from Java) to decipher which branch is selected when. That's why a clear `if` statement is a much preferred solution.